cli/src/commands/git.rs

```diff
@@ -964,21 +965,18 @@ fn cmd_git_push(
     for (branch_name, update) in &branch_updates {
         match (&update.old_target, &update.new_target) {
             (Some(old_target), Some(new_target)) => {
-                if force_pushed_branches.contains(branch_name) {
-                    writeln!(
-                        ui.status(),
-                        "  Force branch {branch_name} from {} to {}",
-                        short_commit_hash(old_target),
-                        short_commit_hash(new_target)
-                    )?;
-                } else {
-                    writeln!(
-                        ui.status(),
-                        "  Move branch {branch_name} from {} to {}",
-                        short_commit_hash(old_target),
-                        short_commit_hash(new_target)
-                    )?;
-                }
+                // TODO(ilyagr): Add color. Once there is color, "Move branch ... sideways" may
+                // read more naturally than "Move sideways branch ...". Without color, it's hard
+                // to see at a glance if one branch among many was moved sideways (say).
+                // TODO: People on Discord suggest "Move branch ... forward by n commits",
+                // possibly "Move branch ... sideways (X forward, Y back)".
+                writeln!(
+                    ui.status(),
+                    "  Move {dir} branch {branch_name} from {old} to {new}",
```

Comment:
nit: it's probably better to map `dir` to `Forward|Backward|Sideways` enum.

We should avoid constructing English sentence in this way.

Reply:
I'm not sure about changing it now; this code will likely be updated a few times in the relatively near future anyway. I think your suggestion will happen naturally if we want to print any more details in the future, but is it worth the extra 10 lines now?

Reply:
I personally wouldn't mind adding two more match arms there, but I agree it can be addressed later. I think it's also good to split the last commit to new PR.

FWIW, I would use something like `{branch} (added|deleted|moved forward|...): {old} -> {new}` to keep the formatting code small. I think `jj git fetch` uses a similar formatting.

Reply:
>  I think it's also good to split the last commit to new PR.

Sure, I'll do that. I rushed the commit a bit because I don't like printing misleading "Force...", but we have more than a week until the next release, which should be plenty of time.

> FWIW, I would use something like {branch} (added|deleted|moved forward|...): {old} -> {new} to keep the formatting code small. I think jj git fetch uses a similar formatting.

This SGTM. For later, I guess it could become "{branch} moved forward {n}" or "{branch} moved sideways, back {n} and forward {m}".

Reply:
Yet another PR I hope to do before the next release is documenting this behavior, but it's OK if that slips to after the release.

Reply:
I haven't edited the commit yet, but I moved it to #3765